packages/eslint-config-typescript-react/src/index.ts

```diff
@@ -4,25 +4,23 @@ require('@rushstack/eslint-patch/modern-module-resolution');
 
 const config: Linter.Config = {
   extends: [
-    require.resolve('eslint-config-airbnb/rules/react'),
-    require.resolve('eslint-config-airbnb/rules/react-a11y'),
-    require.resolve('eslint-config-airbnb/hooks'),
     require.resolve('@rightcapital/eslint-config-typescript'),
+    require.resolve('eslint-config-airbnb/hooks'),
+    require.resolve('eslint-config-airbnb/rules/react-a11y'),
     'plugin:@rightcapital/recommended-react',
+    'plugin:@eslint-react/recommended-legacy',
   ],
   plugins: ['@rightcapital'],
   reportUnusedDisableDirectives: true,
   rules: {
     'jsx-a11y/anchor-is-valid': 'off',
     'jsx-a11y/click-events-have-key-events': 'off',
-    'react/function-component-definition': 'off',
-    'react/jsx-filename-extension': ['warn', { extensions: ['.jsx', '.tsx'] }],
-    'react/jsx-key': ['error'],
-    'react/jsx-no-useless-fragment': ['error', { allowExpressions: true }],
-    'react/jsx-uses-react': 'off',
-    'react/react-in-jsx-scope': 'off',
-    // We are using TypeScript, so it's not necessary to use React's defaultProps
-    'react/require-default-props': 'off',
+    '@eslint-react/naming-convention/filename-extension': [
+      'warn',
+      { extensions: ['.jsx', '.tsx'] },
+    ],
+    '@eslint-react/no-missing-key': ['error'],
```

Comment:
This rule is already included in the `plugin:@eslint-react/recommended-legacy` preset. May we omit it?

https://github.com/Rel1cx/eslint-react/blob/3e3af56a1df933180e423a777887e66f72d2fcf0/packages/plugins/eslint-plugin/src/index.ts#L108